helpers/calendar/analytics_helper.py

```diff
@@ -108,6 +108,11 @@ def get_event_details(self, event, calendar_id):
                     event_details["roomName"] = resource.get(
                         'displayName') or None
                     event_details["summary"] = event.get("summary")
+        else:
+            if event.get('organizer'):
```

Comment:
This could work with elif instead of separating them like this

Reply:
Done